apps/sms/views/conversation/js/compose.js

```diff
@@ -16,14 +16,14 @@
 
 'use strict';
 
-const TYPES = ['image', 'audio', 'video'];
-
 /**
  * Handle UI specifics of message composition. Namely,
  * resetting (auto manages placeholder text), getting
  * message content, and message size
  */
 var Compose = (function() {
+  const TYPES = ['image', 'audio', 'video'];
```

Comment:
Reading my comments again :) There were vanished after rebasing....

Just moved here to not pollute global scope and IIRC FxDesktop was complaining about redefining of constant somehow.